packages/flutter/test/material/checkbox_theme_test.dart

```diff
@@ -70,7 +70,7 @@ void main() {
     expect(description[3], 'overlayColor: MaterialStateProperty.all(Color(0xfffffff2))');
     expect(description[4], 'splashRadius: 1.0');
     expect(description[5], 'materialTapTargetSize: MaterialTapTargetSize.shrinkWrap');
-    expect(description[6], 'visualDensity: VisualDensity#00000(h: 0.0, v: 0.0)');
+    expect(description[6], matches(RegExp(r'^visualDensity: VisualDensity#[0-9a-f]{5}\(h: 0.0, v: 0.0\)')));
```

Comment:
we have a function that strips the hashcodes that you should use here

Reply:
I can't seem to find it. Could you help me out? Thanks.

Reply:
yeah I couldn't find it either. :-)

Ok, found it. `equalsIgnoringHashCodes` is what I was thinking of.